pkg/controller/workspace/provision/pvc.go

```diff
@@ -0,0 +1,72 @@
+package provision
```

Comment:
Please add license headers

Reply:
I ended up using [addlicense](https://github.com/google/addlicense) plus a license template to add headers to my rework PR -- perhaps we should include the header in the repo itself to ease this process/automate a check.

Reply:
+1 to that

Reply:
:tada: https://github.com/che-incubator/che-workspace-operator/pull/54